include/ignition/gui/Application.hh

```diff
@@ -46,6 +46,10 @@ namespace ignition
     class MainWindow;
     class Plugin;
 
+
+    /// /brief Import path for ign-gui QML modules added to the Qt resource system
+    const char *const QML_QRC_IMPORT_PATH = "qrc:/ign-gui-qml/";
```

Comment:
Can this be a member variable to `ApplicationPrivate`, or do we expect downstream applications to need it?

Reply:
I was thinking about downstream applications when I made this a constant.

Reply:
Got it. I think it's best to avoid the global variable though. How about one of these:

1. making it a private variable and adding an accessor function (i.e. `Application::QmlQrcImportPath`
2. creating a global function such as `ignition::gui::qmlQrcImportPath()`, maybe in `Helpers.hh`?

I think it would also be good to use a more modern type to hold this value. How about an `std::string`? I can also see the case for `QString` since that's what `addImportPath` takes.

Reply:
I changed it to a global helper function in 8f4601832e07a1d95bff84bda3c0ba57b7a0c7d7